lib/index.js

```diff
@@ -3,15 +3,11 @@ import fs from 'fs';
 
 fetch('https://www.reddit.com/r/pics/search.json?q=programming&sort=new')
   .then(res => res.json())
-  .then(res => {
+  .then((res) => {
     const data = JSON.stringify(res);
 
-    fs.writeFile('data.json', data, (err) => {
-      if (err) throw err;
-
-      console.log('Created Successfully');
-    });
+    fs.promises.writeFile('data.json', data)
+      .then(() => console.log('Saved Successfully'))
```

Comment:
The primary benefit of promises is that they allow us to avoid "callback hell". Part of what makes callbacks frustrating to work with is their nested structure.

Can we refactor this code so we only have only level of `.then` chains?